### PR TITLE
Update some libraries

### PR DESCRIPTION
**Pull request by SimonSapin**

### CODE REVIEW

**Comment by bjz** [comment]
@SimonSapin build failiure? :/

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "glutin"
-version = "0.4.2"
+version = "0.4.3"
```
Comment (bjz) [inline Cargo.toml]: Not sure what @tomaka's policy regarding API changes, but this alters some public type signatures. Should it be a bump to v0.5.0?

Comment (adambadawy) [inline Cargo.toml]: Which types were changed? I only see fully-qualified names changed to imports.

**Comment by SimonSapin** [comment]
Travis is now green, and I’ve push a blind attempt at fixing AppVeyor. (I don’t have a Windows dev machine)

**Comment by SimonSapin** [comment]
Could someone on Windows take it from here?

File: Cargo.toml (hunk 1 above)
Comment (bjz) [inline Cargo.toml]: They have moved from the `libc` definitions to those in `std::raw`.

Comment (adambadawy) [inline Cargo.toml]: Oops. I read too quickly.

Comment (tomaka) [inline Cargo.toml]: No public API has been changed as far as I can see.

**Comment by tomaka** [comment]
I finished this in #667

**Comment by ozkriff** [comment]
:metal:

File: Cargo.toml (hunk 1 above)
Comment (bjz) [inline Cargo.toml]: Oh were those in private modules? My bad.

**Pull request closed without merging**